frontend/src/index.js

```diff
@@ -3,10 +3,15 @@ import ReactDOM from 'react-dom';
 import './index.css';
 import App from './App';
 import reportWebVitals from './reportWebVitals';
+import GlobalStyle from "./components/GlobalStyle";
+import {BrowserRouter as Router} from "react-router-dom";
 
 ReactDOM.render(
   <React.StrictMode>
+      <Router>
+      <GlobalStyle/>
     <App />
+      </Router>
```

Comment:
Maybe move `<Router>, <GlobalStyle/ >` to your `<App />`component as this will be the place where finally the party happens.

Reply:
Done, thank you!